server/controller/src/main/java/ai/starwhale/mlops/domain/job/ModelServingService.java

```diff
@@ -126,16 +140,23 @@ public ModelServingVo create(
                 .modelVersionId(model.getId())
                 .jobStatus(JobStatus.CREATED)
                 .resourcePool(resourcePool)
+                .lastVisitTime(new Date())
                 .build();
 
-        modelServingMapper.add(entity);
+        long id;
+        synchronized (this) {
+            modelServingMapper.add(entity);
```

Comment:
Is it more clearly with the name of 'modelServingMapper.insertIgnore(...)'?

Reply:
Done in 8f65c60